framework/src/modules/http_api/helpers/filter_peers.js

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright © 2018 Lisk Foundation
```

Comment:
can we put this file other than helper? 🙏

Reply:
Where should I keep it in your opinion, because I kept it here after discussing with @jondubois and I'm not sure exactly where it should reside, maybe in chain's `submodules`? But it already has `peer` file

Reply:
mmm definitely not in chain, but lets keep it here and think the strategy of folder structure later.